### ISSUE TITLE
Imagenet experiment result and pretrained model

### ISSUE THREAD

**Issue by lufo816** [farmingyard/caffe-mobilenet_v2#1]
Thanks for your network prototxt. May you provide Imagenet experiment result and pretrained model? I think it will be helpful.

**Comment by farmingyard** [comment]
@lufo816 Still training, the result is not better yet.

**Comment by xuguozhi** [comment]
What is the current top1 val accuracy achieved?

**Comment by FatherOfHam** [comment]
maybey you can try to transfer the model trained in pytorch to caffe.

**Comment by farmingyard** [comment]
@FatherOfHam 
Yes, the pytorch result is good, but my caffe result is only 70% more or less.
